blueoil/converter/templates/include/global.tpl.h

```diff
@@ -31,12 +31,23 @@ limitations under the License.
 #endif
 using QUANTIZED_PACKED_KERNEL = QuantizedPacked<{{ params.default_qword_dtype.cpptype() }}>;
 
+#ifdef AARCH32
 #define IP_CSR_ADDR 0xFF200000
-#define INPUT_ADDR 0x20000000
-#define OUTPUT_ADDR 0x28000000
-#define KERNEL_ADDR 0x38000000
-#define THRESHOLD_ADDR 0x3F000000
+#define HW_BASE_ADDR 0x20000000
+#else
+#define IP_CSR_ADDR 0xB0000000
+#define HW_BASE_ADDR 0x1F000000
+#endif
+
+#define INPUT_OFFSET 0x00000000
+#define OUTPUT_OFFSET 0x08000000
+#define KERNEL_OFFSET 0x18000000
+#define THRESHOLD_OFFSET 0x1F000000
 
+#define INPUT_ADDR (HW_BASE_ADDR+INPUT_OFFSET)
```

Comment:
`INPUT_OFFSET ` is zero. Can we use

```
#define INPUT_ADDR HW_BASE_ADDR
```

and remove `INPUT_OFFSET` can be removed?

Reply:
It's better to keep `INPUT_OFFSET` because the value of `INPUT_OFFSET` might be changed.
And I would like to use same manner as others like `OUTPUT_OFFSET`.

Reply:
OK. Then no problem to keep it.